proposals/0314-async-stream.md

```diff
@@ -114,8 +114,8 @@ extension QuakeMonitor {
       monitor.quakeHandler { quake in
         continuation.yield(quake)
       }
-      monitor.onTermination = { _ in 
-        monitor.stopMonitoring
+      continuation.onTermination = { _ in 
+        monitor.stopMonitoring // also retains the monitor, preventing deinit
```

Comment:
This comment doesn't seem like something that needs to be in a proposal; it's not explaining anything important to readers.
```suggestion
        monitor.stopMonitoring()
```

I assume we do need to call `stopMonitoring`, though.